src/routes/(console)/organization-[organization]/change-plan/+page.svelte

```diff
@@ -335,14 +361,29 @@
             {/if}
         </Form>
         <svelte:fragment slot="aside">
-            {#if billingPlan !== BillingPlan.FREE && $organization.billingPlan !== billingPlan && $organization.billingPlan !== BillingPlan.CUSTOM}
-                <EstimatedTotalBox
+            {#if billingPlan !== BillingPlan.FREE && $organization.billingPlan !== billingPlan && $organization.billingPlan !== BillingPlan.CUSTOM && isUpgrade}
+                <EstimatedTotal
+                    bind:error={error}
+                    bind:billingBudget={billingBudget}
+                    organizationId={$organization.$id}
                     {billingPlan}
                     {collaborators}
-                    bind:couponData
-                    bind:billingBudget
-                    {isDowngrade} />
+                    {couponId} />
             {:else if $organization.billingPlan !== BillingPlan.CUSTOM}
+                {#if isDowngrade}
+                    <section
+                        class="card u-flex u-flex-vertical u-gap-8"
+                        style:--p-card-padding="1.5rem"
+                        style:--p-card-border-radius="var(--border-radius-small)">
+                        <slot />
```

Comment:
Let's use the `Card` component... also why is there a `<slot />`

Reply:
updated.